src/lib/authserver/ui/new_dir_inst.rb

```diff
@@ -120,7 +120,7 @@ def ok_handler
     # Always remove the ini file
     DS389.remove_setup_ini
     if !ok
-      Popup.Error(_('Failed to set up new instance! Log output may be found in /var/log/YaST/y2log'))
+      Popup.Error(_('Failed to set up new instance! Log output may be found in /var/log/YaST2/y2log'))
```

Comment:
Ideally the path should be moved out from the translatable texts and shared to avoid duplication:

```ruby
def y2log
   "/var/log/YaST2/y2log"
end

Popup.Error(format(_("Failed to set up new instance! Log output may be found in %{y2log}"), y2log: y2log))


```

Reply:
> @Firstyear,
> 
> First of all, thank you :)
> 
> Although the PR has been out of our radar for a couple of months, I think we can resume it and merge to master now. Are you willing to follow @lslezak's suggestions before continuing?
> 
> Regards.

Sure, but won't that mean I need to update every translation as well?